### ISSUE TITLE
Packaging request: IOTA wallet

### ISSUE THREAD

**Issue by jluttine** [NixOS/nixpkgs#32644]
This is just a placeholder for IOTA wallet packaging request: https://github.com/iotaledger/wallet

Feel free to comment here if you start working on it.

**Comment by oyren** [comment]
I was trying to package it but it did't work, had some problems with Electron.

At the moment there is a new IOTA wallet in development called [Trinity](https://blog.iota.org/iota-trinity-wallet-next-steps-12922a53f8d2), so i would wait for this.

**Comment by wamserma** [comment]
IOTA wallet is depracted since more than a year. @jluttine I think this can be closed.

**Comment by oyren** [comment]
The new Iota wallet is called Trinity (as mentioned before). It is now available and can be found [here](https://github.com/iotaledger/trinity-wallet). Perhaps a new Packaging request should be created.

**Comment by ivantomac** [comment]
I only just noticed this ticket. I've had the IOTA wallet packaged for a while now and just recently I've also packaged the Trinity wallet, as well as a couple of other crypto wallets such as the NEON wallet and the Avalanche wallet.

I could submit a patch.

They're all missing icons and so just show up with a generic icon, not sure if that's an issue for inclusion in NixOS packages.
The Avalanche wallet is a bit of a hack as it just runs a python http server on a random port and launches a Chromium instance in app mode with a temporary directory for user data and all extensions disabled.
